pkg/agent/types/net.go

```diff
@@ -35,4 +42,7 @@ const (
 	// Each distinct subnet uses one route table. 20 subnets should be enough.
 	MinEgressRouteTable = 101
 	MaxEgressRouteTable = 120
+
+	// EgressPolicyRouteTable is the route table ID which is used to add policy routing rules in hybrid mode.
+	EgressPolicyRouteTable = 121
```

Comment:
I would also suggest using a larger value so there is a bit of a gap, maybe 141?